### PR TITLE
Define a logging facade and use winston

### LINKED ISSUE #7: [Feature]: Add logger utility to consolidate and remove console usages
I'm imagining either a new `utils/logger-utils.js` file with some functions for logging `info`/`warnings`/`errors`, could probably even use chalk to make it look nicer in the console.

https://www.npmjs.com/package/chalk

Eventually, I'd like to move to passing errors to rollbar or something like it so a basic implementation would be a good start.

### PR DESCRIPTION
**Pull request by christianfds**
## Issue

- Resolves #7 

## Testing Steps

1. Run any command to see the logging in action. Or you can run `npm run test:logger`

## Additional Information

1. You can add a transport layer on winston to handle rollbar though it.

(Sorry it took a while to send the PR, just go around to doing it this afternoon)

### CODE REVIEW

File: src/utils/log-utils.ts [hunk 1]
```diff
@@ -12,18 +14,66 @@ function getLogger() {
...
+        }
+        return '';
+    };
 
-    return console;
+    return winston.createLogger({
+        level: 'info',
+        transports: [
+            new winston.transports.Console({}),
+            // TODO: Evalute if we want to add rollbard a transport from winston or call it from the facade
```
Comment (seriouslysean) [inline src/utils/log-utils.ts]:
```suggestion
            // TODO: Evalute if we want to add Rollbar a transport from Winston or call it from the facade
```

**Review by seriouslysean — COMMENTED** [review]
This looks pretty straight forward, thanks for putting it together!

I don’t know much about Winston but it seems like a transport is the way to go for using Rollbar with it.

https://github.com/GorillaStack/winston-rollbar

Is there anything Winston is offering that console wasn’t that’s really nice to have?

**Comment by christianfds** [comment]
Yes! I didn't add it cause I don't know a lot about the different transports available for this integration.

Winston is a widely supported logging lib in the javascript community, i really liked how it provides the same capabilities of the default logging lib from python, which allows you to handle a single log event in multiple levels and different ways. So if you want, for example, to:
- Route every error log to Rollbar you can add a transport layer that handles only error logs;
- Have a debug transport that logs to a rotating file on disk, you can add a transport to disk;
- Have specific logging being sent to Cloudwatch/Loggly/New Relic or any other logging service available, you most certainly will have a transport ready to use. As each transport can have its own configuration, you can log in different ways to each transport.

(As you can see I really liked the transport settings xP)

Since it provides all these available configurations, it is better than the `console.*` for complex/sensitive projects. And the formatting really makes it easier to read the logs, either on console or anywhere else.

**Review by seriouslysean — APPROVED** [review]
@christianfds awesome, this is so good! I’ll take a look in to Winston and see where to go from here. Thanks so much for the great explanation and of course for doing the work, very much appreciated!

**Pull request merged**